src/features/common/Button.tsx

```diff
@@ -27,6 +28,7 @@ const buttonVariants = tv({
     },
     size: {
       small: "text-xs py-1 px-3",
+      none: "p-0 -mt-1.5",
```

Comment:
`size="none"` is a bit weird - would have gone with `tiny`, `xsmall`, or `smallest` probably.

Reply:
I changed it to `smallest`. Will squash and merge